src/lib/debug/debug_uart.c

```diff
@@ -285,6 +286,92 @@ static BaseType_t debugSerialCommand(char *writeBuffer,
       // Transmit bytes
       // should have generic tx task for all interfaces?
 
+    } else if (strncmp("rxLevel", parameter, parameterStringLength) == 0) {
+
+      parameter = FreeRTOS_CLIGetParameter(
+                      commandString,
+                      2, // Get the second parameter (serial interface)
+                      &parameterStringLength);
+      SerialHandle_t *handle = serialFindInterface(parameter, parameterStringLength);
+
+      if (handle == NULL) {
+        printf("ERR Invalid interface\n");
+        break;
+      }
+
+      parameter = FreeRTOS_CLIGetParameter(
+                      commandString,
+                      3, // Get the rest of the parameters
+                      &parameterStringLength);
+
+      LL_USART_Disable((USART_TypeDef *)handle->device);
+
+      if (strncmp("high", parameter, parameterStringLength) == 0) {
+        LL_USART_SetRXPinLevel((USART_TypeDef *)handle->device, LL_USART_RXPIN_LEVEL_STANDARD);
+      } else if (strncmp("low", parameter, parameterStringLength) == 0) {
+        LL_USART_SetRXPinLevel((USART_TypeDef *)handle->device, LL_USART_RXPIN_LEVEL_INVERTED);
+      } else {
+        printf("ERR Invalid paramters\n");
```

Comment:
If we break here, will the USART be disabled indefinitely until someone tries again?
Maybe we should just re-enable with the current config if we have a invalid CLI params

Reply:
oops yeah  - i'll fix that